### PR TITLE
Feature/economy administer deposits

### PR DESCRIPTION
**Pull request by alexaor**
New PR since I completely botched up the old one and it was less work to checkout from an old commit instead of fixing that nightmare of a rebase, appending old post as its still valid.

Opening up a WIP PR seeing as I have some doubts about the way I went about designing the views and tests, although I can't seem to be able to put exactly what the issue is into a sentence.

The deposit_detail view feels a bit clunky and could probably be refactored,  could maybe also apply to other functions in views.py.

Otherwise missing things so far
- [x] View for editing a Deposit
- [x] Unit tests for view to edit Deposit
- [ ] Error handling in views
- [ ] Tests for error handling

### CODE REVIEW

File: economy/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
-# -*- coding: utf-8 -*-
-# Generated by Django 1.11.4 on 2017-11-09 18:29
-from __future__ import unicode_literals
+# Generated by Django 2.2 on 2019-06-01 18:07
```
[the remark this answers is not in the document]
Comment (alexaor) [reply economy/migrations/0001_initial.py]: Had issues with migrating and eventually got so frustrated i took the nuclear option. Intention was to checkout migrations from develop and try to remedy it, though I admit I forgot I even did that at this point. Good catch.

File: economy/forms.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from django import forms
+from economy.models import Deposit, DepositComment
+
+
+class DepositForm(forms.ModelForm):
+    class Meta:
+        model = Deposit
+        fields = ('amount', 'receipt', 'description', 'account')
+        exclude = ('account',)
```
[the remark this answers is not in the document]
Comment (alexaor) [reply economy/forms.py]: As I wrote this I throught all fields had to be included then excluded as you wanted to limit what was rendered. Fixed now, in addition to same error in DepositCommentForm

File: economy/tests/tests.py [hunk 3]
```diff
@@ -115,3 +121,269 @@ def test__long_comment_value__renders_ellipses(self):
...
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.deposit_comment = DepositCommentFactory()
+
+    def test__valid_deposit_comment_form__is_valid_returns_true(self):
+        form = DepositCommentForm(data={'comment': self.deposit_comment.comment})
+        self.assertTrue(form.is_valid())
+
+    def test__invalid_deposit_comment_form__is_valid_returns_false(self):
```
[the remark this answers is not in the document]
Comment (alexaor) [reply economy/tests/tests.py]: Semantics, its an invalid form and the test is asserting if is_valid returns false although I agree with the ambiguous nature of the test name. Will rename it.

**Review by trmdy — APPROVED** [review]
Looks good. Lacking some design, but we'll take that in another PR.

**Pull request merged**